.gitignore

```diff
@@ -26,3 +26,5 @@ app/assets/images
 config/env.yml
 
 queue_listeners/config/env.yml
+
+.DS_Store
```

Comment:
I'm actually not sure why I asked this question. I know what the DS_Store files are, and why they should be in gitignore, so I'm going to blame it on my brain taking a temporary holiday. :)